slsDetectorServers/ctbDetectorServer/slsDetectorFunctionList.c

```diff
@@ -480,10 +475,7 @@ void setupDetector() {
     naSamples = 1;
     ndSamples = 1;
 #ifdef VIRTUAL
-    virtual_status = 0;
-    if (isControlServer) {
```

Comment:
Was this check redundant, or is now done somewhere else?

Reply:
Its not needed anymore. Before we had local variables or accesssing via files. Now both servers access via shared memory.